### PR TITLE
Added unit tests

### PR DESCRIPTION
**Pull request by visakh-ms**

### CODE REVIEW

File: jest.config.ts [hunk 1]
```diff
@@ -37,7 +37,6 @@ export default {
   coverageReporters: [
     //   "json",
     'text',
-    'cobertura',
```
Comment (Tezra) [inline jest.config.ts:40]: We need the cobertura and junit test results.

File: src/ExportPDFPlugin.test.ts [hunk 2]
```diff
@@ -1,14 +1,108 @@
...
+    });
+    const view = new EditorView(
+      {mount: dom},
+      {
+        state: state,
+      }
+    );
+    jest.spyOn(exportp, 'exportPdf').mockImplementation(exportPdfMock);
+    plugin.perform(view);
+    expect(executeMock).not.toHaveBeenCalledWith(undefined, undefined, view);
```
Comment (Tezra) [inline src/ExportPDFPlugin.test.ts:89]: executeMock is never used in this test.

File: src/exportPdf.test.ts [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+      schema: effSchema,
+    });
+    const view = new EditorView(
+      {mount: dom},
+      {
+        state: state,
+      }
+    );
+    exportPdf.exportPdf(view);
+    expect(mockPdfObject.save()).toEqual('2023-06-09_09:18:09.pdf');
```
Comment (Tezra) [inline src/exportPdf.test.ts:75]: expect only tests mock behavior. Check that work has actually been done.

File: src/exportPdf.test.ts [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+      state: {
+        doc: {
+          attrs: {
+            objectId: '0001-2365-4312-0567',
+          },
+        },
+      },
+    } as unknown as EditorView;
+    exportPdf.exportPdf(mockView);
+    expect(mockPdfObject.save()).toEqual(mockView.state.doc.attrs.objectId);
```
Comment (Tezra) [inline src/exportPdf.test.ts:102]: expect only tests mock behavior. Check that work has actually been done.

**Review by Tezra — CHANGES_REQUESTED** [review]

**Review by Tezra — APPROVED** [review]

**Review by rodgersgb — APPROVED** [review]

**Pull request merged**